package.json

```diff
@@ -181,7 +178,6 @@
     "istanbul": "^0.4.4",
     "javascript-state-machine": "^3.1.0",
     "jsonwebtoken": "^8.0.1",
-    "jszip": "^3.1.5",
     "karma": "^2.0.0",
     "karma-browserify": "^5.1.3",
     "karma-chrome-launcher": "^2.0.0",
```

Comment:
The above dependences were specific to the report download. This functionality has been moved to a standalone app. Hence the dependencies are no longer required.

Reply:
Removing things is always welcome 😉